src/flick/friend/views.py

```diff
@@ -113,8 +113,8 @@ def _create_accepted_notification(self, from_user, to_user):
         to_user = Profile.objects.get(user=to_user)
         notif = Notification()
         notif.notif_type = "accepted_request"
-        notif.from_user = from_user
-        notif.to_user = to_user
+        notif.from_user = to_user
+        notif.to_user = from_user
```

Comment:
a litto confused by this because i thought the meaning of `from_user` and `to_user` is `from_user` made a friend request to `to_user` but  if we flip `to_user` and `from_user` doesn't it  change that relationship?